Package.swift

```diff
@@ -125,16 +137,52 @@ var targets: [PackageDescription.Target] = [
         ]),
 ]
 
+#if swift(>=5.7.1)
```

Comment:
Just drop this conditional completely. We are going to merge this PR after we drop 5.6 and then we will just make `5.7.1` or bottom version.

Reply:
Fixed